python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -261,17 +261,17 @@ def __init__(self, model: Any) -> None:
         # Set eval automatically for user for batchnorm and dropout layers
         self._model.eval()
 
-    def predict(self, dataset, detection_error_threshold=0.1):
+    def predict(self, dataset, iou_threshold=0.1):
         """
         Prediction function for object detector.
 
         :param dataset: The dataset to predict on.
         type dataset: PIL Image or numpy.ndarray
         param device: device pytorch is writing to
         type device: pytorch device
-        param detection_error_threshold: amount of acceptable error.
+        param iou_threshold: amount of acceptable error.
```

Comment:
Perhaps a bit of additional description on what IoU is?

Reply:
Good idea! Added a description to predict, predict_proba, and .md file.